types/denom.go

```diff
@@ -0,0 +1,80 @@
+package types
+
+import (
+	"fmt"
+	"math/big"
+)
+
+// Denomination constants
+const (
+	Uatom = "uatom"
+	Matom = "matom"
+	Atom  = "atom"
+)
```

Comment:
These denomination constants (and `denomUnits`)  should only exist as a part of gaia and be registered from there - they ought to be removed from the `SDK`

Reply:
Hmm agreed. I removed the constants and their units in the map.

Reply:
What's going to happen to the `DefaultBondDenom` defined in types/staking.go then?

Reply:
What do you mean @alessio?

Reply:
@alessio Yeah maybe we should remove `DefaultBondDenom` too make gaia and tests explicitly define the denom

Reply:
I think we should, but it's out of scope of this PR 👍